### DISCUSSION TITLE
only for groups ot for individuals as well?

### DISCUSSION THREAD

**Discussion by DoctorSubtilis** [42wim/matterbridge#1468] [Help / Q&A] [answered]
**Describe the bug**
This is a question, not a bug report: so far I managed to connect whatsapp groups, but it is possible connect individuals as well?
And, if so, how?
Thanks

**Environment (please complete the following information):**
 - OS: KDE Neon linux
 - Matterbridge version: 1.22.1

**Comment by qaisjp** [comment, marked as answer]
If you mean like direct messaging: this is not possible. You can follow https://github.com/42wim/matterbridge/issues/191 for more information.

**Reply by DoctorSubtilis** [reply]
thank you
